### PR TITLE
[fix][broker] Fix AvgShedder strategy check

### PR DESCRIPTION
**Pull request by Demogorgon314**
### Motivation

The check for AvgShedder strategy is wrong, we should use `loadBalancerLoadPlacementStrategy` instead of `loadBalancerPlacementStrategy`.

The `loadBalancerPlacementStrategy ` is used by `SimpleLoadManagerImpl`, and it is deprecated.

### Modifications

Use `loadBalancerLoadPlacementStrategy` instead of `loadBalancerPlacementStrategy` to check the `AvgShedder `strategy

### Documentation

- [ ] `doc` 
- [ ] `doc-required` 
- [x] `doc-not-needed` 
- [ ] `doc-complete`

### CODE REVIEW

**Review by heesung-sn — APPROVED** [review]
Do you know Why we have these two similar configs? This is confusing indeed.

**Comment by Demogorgon314** [comment]
@heesung-sn The `loadBalancerPlacementStrategy ` is used by `SimpleLoadManagerImpl`, and it is deprecated.

**Review by thetumbled — APPROVED** [review]
LGTM.

**Comment by thetumbled** [comment]
This bug fix is not included into branch 3.0 and 3.3, we need to cherry pick it into 3.0 and 3.3 and wait for next release. @heesung-sn @Demogorgon314 @lhotari

**Comment by lhotari** [comment]
> This bug fix is not included into branch 3.0, 3.2 and 3.3, we need to cherry pick it into 3.0 and 3.3 and wait for next release. @heesung-sn @Demogorgon314 @lhotari

The person merging the PR should ensure that the release labels are on the PR before it gets merged. That's how the release manager knows what it planned for a release. @thetumbled Please find the information in this email: https://lists.apache.org/thread/ryyfv8o33yyw9hmo5gnxn71boocn3mzs . This information is currently missing from our contribution guide.

**Comment by lhotari** [comment]
@thetumbled The cherry-picking process is now documented in https://pulsar.apache.org/contribute/maintenance-process/

**Pull request merged**
